docs/evaluate/temporal-cloud/gcp-private-service-connect.mdx

```diff
@@ -11,22 +11,34 @@ keywords:
   - security
   - temporal cloud
   - gcp
+  - google cloud
 tags:
   - security
   - temporal-cloud
   - gcp
+  - google cloud
   - private service connect
   - private-connectivity
 ---
 
-#### GCP Private Service Connect
+#### Google Cloud Private Service Connect
 
-[GCP Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) allows you to open a path to Temporal without opening a public egress.
+[Google Cloud Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) allows you to open a path to Temporal without opening a public egress.
 It establishes a private connection between your Google Virtual Private Cloud (VPC) and Temporal Cloud.
 This one-way connection means Temporal cannot establish a connection back to your service.
 This is useful if normally you block traffic egress as part of your security protocols.
 If you use a private environment that does not allow external connectivity, you will remain isolated.
 
+:::note
+
+If you are interested in leveraging Google Cloud Private Service Connect in your Namespaces, [create a support ticket](/cloud/support#support-ticket) that includes the following information:
+
+- Google Cloud Region: The Region in which your connection will go through.
```

Comment:
We don't require the region, because the consumer must create the endpoint within the same region as the producer (e.g. temporal cloud namespace region). To enable cross-region connectivity within their VPC, they can enable Global Access on their consumer endpoint if they have the `GLOBAL` routing mode configured on their VPC.

Reply:
You are right, and AWS now has a global option too which we should add to the documentation on the AWS side. That being said, as mentioned in the GCP call, I think for now it's better to keep asking the same information no matter for which cloud as it standardizes things across. Once we get to self-service with PCv3 we won't need any of those informations anymore and can clean things up!!